sdk/tests/conformance/extensions/webgl-compressed-texture-etc.html

```diff
@@ -140,11 +140,12 @@
 
     debug("");
     if (contextVersion >= 2) {
+      var expectedError = isPositive ? gl.INVALID_OPERATION: [gl.INVALID_ENUM, gl.INVALID_OPERATION];
```

Comment:
It seems a little odd to allow two either INVALID_ENUM or INVALID_OPERATION. But the difference may not matter enough to specify and fix.